### ISSUE TITLE
Dock.hide() Throws Error on Windows

### ISSUE THREAD

**Issue by caleblogan** [aaroncox/anchor#1]
In main.dev hide is being called before dock is defined throwing an error. I think this issue is specific to windows. https://github.com/aaroncox/anchor/blob/master/app/main/main.dev.js#L74

**Comment by aaroncox** [comment]
Good catch, that might need to be wrapped in a check for macOS. I had figured that also applied to the taskbar on windows.
